[CUBRIDMAN-187] 11.3 release note (#474)
CUBRID/cubrid-manual: en/release_note/release_note_latest_ver.rst

Overview
========

CUBRID 11.3 is the latest stable version that includes new features, significant changes and enhancements.

.. TODO: UPDATE WITH DETAILS.

CUBRID 11.3

> * improves connectivity by improving database links.

Reviewer: How about changing as the following ?
improves connectivity by enhancing database links.

Author: Reflected it

* is more stable, faster, and more convenient for administrators.
* fixes a large number of critical bugs.
* includes code refactoring and modernization.

> CUBRID 11.3 provided user convenience by providing a table extension format (object@server) for database links, and **improved connectivity** by providing the ability to insert, modify, and delete data as well as query data for remote DB.

Reviewer: ```suggestion
CUBRID 11.3 provided user convenience by providing a table extension style (object@server) for database links, and **improved connectivity** by providing the ability to insert, modify, and delete data as well as retrieve data for remote DB.
```

Author: Reflected it

> CUBRID 11.3 is **faster**. This version further improves performance through improvements to query optimization and aggregation functions (min, max group_concat), such as predicate pushdown, view transformation, and removal of unnecessary join tables. Additionally, performance problems caused by skewed indexes were improved by providing a deduplication option when creating an index (or foreign key).

Reviewer: ```suggestion
CUBRID 11.3 is **faster**. This version further improves performance through improve aggregation functions (min, max group_concat) and query optimization such as predicate pushdown, view transformation, and removal of unnecessary join tables. Additionally, performance problems caused by skewed indexes were improved by providing a deduplication option when creating an index (or foreign key).
```

Author: Reflected it

CUBRID 11.3 **improved administrator convenience** by improving the csql, unloaddb, and loaddb utilities.

> The database volume of CUBRID 11.3 is compatible with the CUBRID 11.2 volume. (However, when a volume created in CUBRID 11.2 is used in CUBRID 11.3 due to a change in view query spec of db_index and db_vclass among catalog views, unexpected results may be obtained when executing a corresponding catalog view query.)

Reviewer: ```suggestion
The database volume of CUBRID 11.3 is compatible with the CUBRID 11.2 volume. (However, when a volume created in CUBRID 11.2 is used in CUBRID 11.3 due to a change in view query spec of db_index and db_vclass among catalog views, unexpected results may be obtained when executing a query related the catalog view.)
```

Author: Reflected it

.. TODO: coming soon 

Driver Compatibility
--------------------

*   The JDBC and CCI driver of CUBRID 11.3 are compatible with the DB server of CUBRID 11.2, 11.1, 11.0, 10.2, 10.1, 10.0, 9.3, 9.2, 9.1, 2008 R4.4, R4.3 or R4.1.
*   To upgrade drivers are highly recommended.

For more details on changes, see the :ref:`11_3_changes`. Users of previous versions should check the :ref:`11_3_changes` and :ref:`11_3_new_cautions` sections.

.. _11_3_changes:

11.3 Changes
============

Please refer to `change logs of CUBRID 11.3 <https://github.com/CUBRID/cubrid/releases/tag/v11.3>`_.

Cautions
========

.. _11_3_new_cautions:

New Cautions
------------

The query optimizer's statistical information is not automatically updated when DDL statement is executed, the user must manually execute UPDATE STATISTICS statement to update the statistical information. (see :doc:`/sql/tuning`)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> When changing the column type of the AUTO_INCREMENT property in the ALTER TABLE statement, it cannot be changed to a type that cannot be used as the AUTO_INCREMENT property. When changing the type of a column where the default value is specified, an error occurs if type conversion of the specified default value to the changed type is not possible. (see :ref:`change-column`)

Reviewer: ```suggestion
When changing the type of a column included the AUTO_INCREMENT property or default value in the ALTER TABLE statement, an error occurs if the changed type cannot have the AUTO_INCREMENT property or is a type whose existing default value cannot be changed. (see :ref:`change-column`)
```

Author: Reflected it

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Unable to get Connection object through cubrid.jdbc.driver.CUBRIDDriver.getDefaultConnection() from server-side JDBC. Instead, you should use DriverManager.getConnection("jdbc\:default\:connection\:"). (see :ref:`jsp-server-side-jdbc-connection`)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> If an invalid index name or table name is specified in the USE INDEX (USING INDEX) statement, it is ignored without processing an error. However, if the parameter error_log_warning is set to yes, it is left in the log file.

Reviewer: ```suggestion
If an invalid index name or table name is used in a USE INDEX (USING INDEX) statement, it will be ignored rather than occuring an error. (To leave it in the log file, set the error_log warning parameter to yes.)
```

Author: Reflected it

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> When writing the CREATE INDEX statement, COMMENT was placed before the WITH clause or INVISIBLE designation, but was changed to be placed at the very end of the statement. (see :doc:`/sql/schema/index_stmt`)

Reviewer: ```suggestion
When writing a CREATE INDEX statement, the position of COMMENT was changed to the end of the statement after the WITH or INVISIBLE clause. (see :doc:`/sql/schema/index_stmt`)
```

Author: Reflected it
